### PR TITLE
Documentation suite

### PR DESCRIPTION
**Pull request by standage**
MicroHapDB's README has long served as a one-stop-shop for documentation. This PR adds a documentation suite (built with sphinx) that has permitted me to fill out the documentation a bit more. The new documentation includes dedicated pages for:

- getting started
- installation
- statistics for ranking markers
- details for configuring target sequences
- brief Q&A with pointers to the dbbuild instructions for adding new markers and/or rebuilding the database from scratch
- CLI reference
- cited works

I've trimmed the README down a bit now that this information is included in dedicated documentation.

I'm attaching a copy of the rendered HTML docs here. Once this PR is merged, I'll configure ReadTheDocs to build and host the documentation automatically, as is done for [MicroHapulator](https://microhapulator.readthedocs.io/).

[docs.zip](https://github.com/bioforensics/MicroHapDB/files/10014724/docs.zip)

### CODE REVIEW

File: microhapdb/cli/marker.py [hunk 1]
```diff
@@ -92,7 +92,6 @@ def display(result, view_format, delta=10, minlen=80, extend_mode=0, trunc=True,
 
 
 def subparser(subparsers):
-    desc = microhapdb.cli.bubbletext + "\nRetrieve marker records by identifier or query"
```
Comment (standage) [inline microhapdb/cli/marker.py:95]: Bubbletext did not render well on the sphinx autodocs. It's still shown when `microhapdb -h` is run, but no longer when e.g. `microhapdb marker -h` is run.

**Comment by standage** [comment]
No code changes to speak of in this PR, but @rnmitchell if you have time, I'd be interested in any feedback you have on the clarity of the documentation. Thanks!

**Comment by standage** [comment]
Good catch. I need to update `make devdeps` to include sphinx, sphinx-argparse, and myst-parser. I'll update the Makefile, but in the meantime you can install those packages with conda or pip.

File: docs/ranking.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+## Informativeness (*I<sub>n</sub>*)
+
+Rosenberg's informativeness for assignment to populations ($I_n$) is computed on 26 global population groups.
+
+
+## Fixation index (*F<sub>ST</sub>*)
+
+The fixation index ($F_{ST}$) is computed on 26 global population groups.
+
+The $I_n$ and F_{ST} statistics measure *between-population* allelic variation at a locus, which corresponds to the marker's utility for predicting population of origin.
```
Comment (rnmitchell) [inline docs/ranking.md:24]: I think this formatting of FST got messed up? Looks like it should be ```$F_{ST}$```?

File: docs/targets.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+microhapdb marker --panel=markerids.txt --delta=25 --min-length=150 --extend-mode=3 \
+    --format=offsets \
+    > marker-definitions.tsv
+```
+
+
+## Impacted outputs
+
+MicroHapDB provides several different views for marker information.
+This includes the default tabular summary format, a detailed human-readable description of the marker, a view for target sequences in Fasta format, and a table of the position of each marker's SNPs.
```
Comment (rnmitchell) [inline docs/targets.md:102]: This is very nitpicky but FASTA not Fasta to be correct :)

**Review by rnmitchell — COMMENTED** [review]
I just found a few small typos but otherwise looks great!

**Comment by standage** [comment]
Good catches! Typos fixed.

**Review by rnmitchell — APPROVED** [review]
Looks great to me!

**Comment by standage** [comment]
Thank you for your careful review!

**Pull request merged**